R/inst/tutorials/07-autopilot/autopilot.Rmd

```diff
@@ -0,0 +1,34 @@
+---
+title: "Episode 7: Autopilot"
+output: html_notebook
+---
+
+**This notebook shows how you can track Metaflow flows that have been scheduled to execute in the cloud.**
+
+## Import the metaflow client
+```{r}
+suppressPackageStartupMessages(library(metaflow))
+message("Current metadata provider: ", metaflow::get_metadata())
+```
+
+## Plot a timeline view of a scheduled run of MovieStatsFlow
+When you triggered your flow on AWS Step Functions using `step-functions trigger`, you would have seen an output similar to - 
+```{bash}
+...
+Workflow MovieStatsFlow triggered on AWS Step Functions (run-id sfn-b1bd0ec2-e29b-4818-90ad-507aa600dbf0).
```

Comment:
replace sfn-b1bd0ec2-e29b-4818-90ad-507aa600dbf0 with sfn-dolor-sit-amet here and in L:24